src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const defaultPrice = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  const price = days * defaultPrice;
+
+  if (days >= 3 && days < 7) {
+    return price - smallDiscount;
+  } else if (days >= 7) {
+    return price - bigDiscount;
+  } else {
+    return price;
+  }
```

Comment:
too complicated, let's simplify it
```suggestion
  if (days >= 7) {
    return price - bigDiscount;
  } 
  
 if (days >= 3) {
    return price - smallDiscount;
  }
  
  ...
```